dask/tests/test_task_spec.py

```diff
@@ -62,6 +62,28 @@ def test_convert_legacy_dsk_skip_new():
     assert converted == dsk
 
 
+def test_repr():
+    t = Task("key", func, "a", "b")
+    assert repr(t) == "<Task 'key' func('a', 'b')>"
```

Comment:
Thoughts on adding an equality sign here?

```suggestion
    assert repr(t) == "<Task 'key'=func('a', 'b')>"
```

Reply:
hmm, I find the `=` doesn't add much clarity. I personally find the `=` even a little confusing since it's broken syntax :P

I suspect you are fine with this for now? I strongly suspect we'll want to revisit this repr at some point. This is, e.g. also logged on the worker exception log message and we might want to tweak it but I would leave it for a follow up

Reply:
yeah fine with this for now

I like the equality sign better, it guides my eye to some degree